### PR TITLE
Freesurfer scripts

### PR DESCRIPTION
**Pull request by vinferrer**
Add scripts for freesurfer preprocessing:
01_freesurfer.sh is the main script executed by singularity
singularity_freesurfer_run.sh is the script that will call the main script with singularity. This is still a work in progress, I just wanna try the main script in a subject

### CODE REVIEW

**Comment by vinferrer** [comment]
@smoia I think this is ready for merge, just remeber i have to do a tiny change before you merge which changing the wdir variable to PJ_MASK2 scripts folder

File: 30.plugins/01.freesurfer/01_freesurfer.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+orders=" -autorecon1 -noskullstrip -s $SUBJ -i $T1 -$option $T2 -sd $SUBJECTS_DIR"
+recon-all $orders
+cp $SUBJECTS_DIR/$SUBJ/mri/T1.mgz $SUBJECTS_DIR/$SUBJ/mri/brainmask.auto.mgz
+cp $SUBJECTS_DIR/$SUBJ/mri/brainmask.auto.mgz $SUBJECTS_DIR/$SUBJ/mri/brainmask.mgz
+recon-all -autorecon2 -autorecon3 -T2pial -sd $SUBJECTS_DIR -s $SUBJ -hippocampal-subfields-T1 -brainstem-structures
+echo  -e "\e[34m +++ =======================================================================\e[39m"
+echo  -e "\e[34m +++ ------------> CONVERTING FREESURFER OUTPUT TO NII and GII <------------\e[39m"
+echo  -e "\e[34m +++ =======================================================================\e[39m"
+mkdir /Data/${SUBJ}/ses-01/atlas
+mri_convert -i ${SUBJECTS_DIR}/sub-001/mri/aparc.a2009s+aseg.mgz -o /Data/${SUBJ}/ses-01/atlas/${SUBJ}_aparc.a2009s+aseg.nii.gz
```
Comment (smoia) [inline 30.plugins/01.freesurfer/01_freesurfer.sh:48]:
```suggestion
mri_convert -i ${SUBJECTS_DIR}/sub-001/mri/aparc.a2009s+aseg.mgz -o /Data/${SUBJ}/ses-01/atlas/${SUBJ}_aparc.nii.gz
```

**Review by smoia — CHANGES_REQUESTED** [review]
LGTM!

File: 30.plugins/01.freesurfer/01_freesurfer.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+orders=" -autorecon1 -noskullstrip -s $SUBJ -i $T1 -$option $T2 -sd $SUBJECTS_DIR"
+recon-all $orders
+cp $SUBJECTS_DIR/$SUBJ/mri/T1.mgz $SUBJECTS_DIR/$SUBJ/mri/brainmask.auto.mgz
+cp $SUBJECTS_DIR/$SUBJ/mri/brainmask.auto.mgz $SUBJECTS_DIR/$SUBJ/mri/brainmask.mgz
+recon-all -autorecon2 -autorecon3 -T2pial -sd $SUBJECTS_DIR -s $SUBJ -hippocampal-subfields-T1 -brainstem-structures
+echo  -e "\e[34m +++ =======================================================================\e[39m"
+echo  -e "\e[34m +++ ------------> CONVERTING FREESURFER OUTPUT TO NII and GII <------------\e[39m"
+echo  -e "\e[34m +++ =======================================================================\e[39m"
+mkdir /Data/${SUBJ}/ses-01/atlas
+mri_convert -i ${SUBJECTS_DIR}/sub-001/mri/aparc.a2009s+aseg.mgz -o /Data/${SUBJ}/ses-01/atlas/${SUBJ}_aparc.nii.gz
```
Comment (smoia) [inline 30.plugins/01.freesurfer/01_freesurfer.sh:48]:
```suggestion
mri_convert -i ${SUBJECTS_DIR}/sub-001/mri/aparc.a2009s+aseg.mgz -o /Data/${SUBJ}/ses-01/atlas/${SUBJ}_aparc.a2009s+aseg.nii.gz
```

**Review by smoia — APPROVED** [review]
LGMT

**Pull request merged**